app/src/main/java/org/oppia/app/application/OppiaApplication.kt

```diff
@@ -4,17 +4,39 @@ import android.app.Application
 import androidx.appcompat.app.AppCompatActivity
 import androidx.multidex.MultiDexApplication
 import org.oppia.app.activity.ActivityComponent
+import org.oppia.domain.oppialogger.exceptions.ExceptionsController
+import org.oppia.domain.oppialogger.exceptions.OppiaUncaughtExceptionHandler
+import org.oppia.util.system.OppiaClock
+import javax.inject.Inject
 
 /** The root [Application] of the Oppia app. */
 class OppiaApplication : MultiDexApplication(), ActivityComponentFactory {
   /** The root [ApplicationComponent]. */
+
+  @Inject
+  lateinit var exceptionsController: ExceptionsController
+  @Inject
+  lateinit var oppiaClock: OppiaClock
```

Comment:
Can we not inject OppiaUncaughtExceptionHandler directly instead of injecting its dependencies and creating it below?

Reply:
Done.